site/__init__.py

```diff
@@ -22,6 +25,19 @@ class Site(Site):
         undefined=StrictUndefined,
     )
 
+    def render_static(self, directory):
+        super().render_static(directory)
+        for file in Path(directory).glob("**/*.css"):
+            pytailwindcss.run(
+                auto_install=True,
+                tailwindcss_cli_args=[
+                    "--input",
+                    f"{file.absolute()}",
+                    "--output",
+                    f"{(self.path / file).absolute()}",
+                ],
+            )
```

Comment:
This section was copied directly from [another repository](https://gitlab.com/jonafato/conferencechats.org/-/blob/520037a87522f0087038686c9ea7b50f597cc72c/src/site.py#L18-29) of mine.